cmd/zt_remove_blob_test.go

```diff
@@ -475,3 +477,109 @@ func (s *cmdIntegrationSuite) TestRemoveBlobsWithDirectoryStubsWithListOfFiles(c
 		c.Assert(len(mockedRPC.transfers), chk.Equals, 0)
 	})
 }
+
+func (s *cmdIntegrationSuite) TestDryrunRemoveSingleBlob(c *chk.C) {
+	bsu := getBSU()
+	containerURL, containerName := createNewContainer(c, bsu)
+	defer deleteContainer(c, containerURL)
+
+	// set up the container with a single blob
+	blobName := []string{"sub1/test/testing.txt"}
+	scenarioHelper{}.generateBlobsFromList(c, containerURL, blobName, blockBlobDefaultData)
+	c.Assert(containerURL, chk.NotNil)
+
+	// set up interceptor
+	mockedRPC := interceptor{}
+	Rpc = mockedRPC.intercept
+	mockedLcm := mockedLifecycleManager{dryrunLog: make(chan string, 50)}
+	mockedLcm.SetOutputFormat(common.EOutputFormat.Text()) //text format
+	glcm = &mockedLcm
+
+	// construct the raw input to simulate user input
+	rawBlobURLWithSAS := scenarioHelper{}.getRawBlobURLWithSAS(c, containerName, blobName[0])
+	raw := getDefaultRemoveRawInput(rawBlobURLWithSAS.String())
+	raw.dryrun = true
+
+	runCopyAndVerify(c, raw, func(err error) {
+		c.Assert(err, chk.IsNil)
+		// validate that none where transferred
+		c.Assert(len(mockedRPC.transfers), chk.Equals, 0)
+
+		msg := <-mockedLcm.dryrunLog
+		//comparing message printed for dry run
+		c.Check(strings.Compare(msg, fmt.Sprintf("DRYRUN: remove %v/%v/", containerURL, blobName[0])), chk.Equals, 0)
```

Comment:
Minor: consider making this validation less tightly coupled, in case the text changes in the future. Perhaps we just need to make sure the keywords are included in the string.